### ISSUE TITLE
Do not assume either highlighting or autosuggestions are installed

### ISSUE THREAD

**Issue by olets** [olets/zsh-abbr#6]
See https://github.com/olets/zsh-abbr/pull/5#issuecomment-590505455

```
❯ c_zsh_abbr_expand_widget:11: command not found: _zsh_highlight
```

**Comment by olets** [comment]
@philwhln this should be fixed in 2.1.2. Let me know!

**Comment by olets** [comment]
Missed a little bit. 2.1.3 removes all references to optional external plugins
